packages/utils/src/account.ts

```diff
@@ -145,6 +157,19 @@ function getWordList(language?: MnemonicLanguages) {
   }
 }
 
+function getAllLanguages() {
+  return [
+    MnemonicLanguages.chinese_simplified,
+    MnemonicLanguages.chinese_traditional,
+    MnemonicLanguages.english,
+    MnemonicLanguages.french,
+    MnemonicLanguages.italian,
+    MnemonicLanguages.japanese,
+    MnemonicLanguages.korean,
+    MnemonicLanguages.spanish,
+  ]
+}
+
```

Comment:
Are you sure we ship with all the word lists? IIRC, we only ship with English and Spanish. If we do, this looks good, but I'd recommend double checking. One trick you could do is to check this at runtime with `Object.keys`, possible with a static list of all possible languages to filter out any object keys that are not languages.

Reply:
Even if we don't have the word lists, having it like this will future-proof the function, if we don't have the word list it will just fail now but if we ever add it in the future we don't have to remember to add it here as well.

I had a types issue using `Object.keys` and other similar options because it detected them as strings rather than the enum value, couldn't find a way of getting all of the values of an enum by googling :(